### ISSUE TITLE
When creating a new Case description field is not displayed on 7.8.18

### ISSUE THREAD

**Issue by sanchezfauste** [salesagility/SuiteCRM#5834]
When you create a new Case the description field is not displayed properly. Blank area is displayed instead HTML editor.

#### Issue

When you create a new Case the description field is not displayed properly. Blank area is displayed instead HTML editor.

The following JS error is throwed:

```js
TypeError: p is not a constructor
TypeError: p.getElement(...) is null
```

#### Expected Behavior

Description field shoud display HTML editor.

#### Actual Behavior

This is how description field is displayed when you create a new case:

![error](https://user-images.githubusercontent.com/10921800/39712605-f1b0ccd6-5223-11e8-87ea-6e5407b67e16.png)

#### Possible Fix

I don't know. QS and JS repairs does not help.

#### Steps to Reproduce

1. Go to Cases module.
2. Click on 'Create Case' link.
3. Observe that `Description` field is not displayed properly and a JS error is throwed.

#### Context

We are trying to create a new Case after upgrade to SuiteCRM 7.8.18 LTS.

#### Your Environment

* SuiteCRM Version used: SuiteCRM 7.8.18 LTS
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Versión 66.0.3359.117 (Build oficial) (64 bits)
* Environment name and version (e.g. MySQL, PHP 7): PHP 7.0.28 and mysql-server-5.7
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 16.04.4 LTS

**Comment by ftoledo** [comment]
I have same error on 7.11.13
Inline edit of description field work. action->edit , html editor fail.
it's a regression bug?

**Comment by chris001** [comment]
> I have same error on 7.11.13
> Inline edit of description field work. action->edit , html editor fail.
> it's a regression bug?

Compare your code file, to the fix file that @PedroErnst added to 7.8, and see if you have the fixed code in your 7.11.13, or no:
https://github.com/salesagility/SuiteCRM/pull/5840/files
